pages.zh/common/chown.md

```diff
@@ -0,0 +1,24 @@
+# chown
+
+> 修改用户和用户组对文件或目录的所有权。
+> 更多信息：<https://www.gnu.org/software/coreutils/chown>.
+
+- 修改文件或目录的所有者：
+
+`chown {{用户}}} {{文件或目录路径}}`
+
+- 修改文件或目录的所有者及所属组：
+
+`chown {{用户}}:{{用户组}} {{文件或目录路径}}`
+
+- 递归修改目录及其子目录和文件的所有者：
+
+`chown -R {{用户}} {{目录路径}}`
+
+- 修改符号链接的所有者：
+
+`chown -h {{用户}} {{符号链接路径}}`
+
+- 修改文件或目录的所有者与参考文件相同：
+
+`chown --reference={{参考文件路径}} {{文件或目录路径}}`
```

Comment:
```suggestion
`chown --reference={{路径/到/参考文件}} {{路径/到/文件或目录}}`
```

Reply:
Thanks for your review, the translations have been fixed.